_includes/header.html

```diff
@@ -11,7 +11,11 @@ <h1 class="brand-heading">&nbsp;</h1>
             <span class="intro-text">{{ content }}</span>
             <h5>&nbsp;</h5>
             <a href="#{{ site.pages_list['About'] }}" class="page-scroll">
-              <img class="img-me" src="{{site.baseurl}}{{ site.me-img }}" alt="">
+              {% if site.me-img contains "https://" %}
```

Comment:
At first i was wondering what happens here ... but I think this is the most feasible way to check if the url is relative or absolute